src/System.Data.OleDb/ref/System.Data.Oledb.cs

```diff
@@ -0,0 +1,472 @@
+        public new System.Data.OleDb.OleDbCommand GetInsertCommand() { throw null; }
+        public new System.Data.OleDb.OleDbCommand GetInsertCommand(bool useColumnsForParameterNames) { throw null; }
+        protected override string GetParameterName(int parameterOrdinal) { throw null; }
+        protected override string GetParameterName(string parameterName) { throw null; }
+        protected override string GetParameterPlaceholder(int parameterOrdinal) { throw null; }
+        public new System.Data.OleDb.OleDbCommand GetUpdateCommand() { throw null; }
+        public new System.Data.OleDb.OleDbCommand GetUpdateCommand(bool useColumnsForParameterNames) { throw null; }
+        public override string QuoteIdentifier(string unquotedIdentifier) { throw null; }
+        public string QuoteIdentifier(string unquotedIdentifier, System.Data.OleDb.OleDbConnection connection) { throw null; }
+        protected override void SetRowUpdatingHandler(System.Data.Common.DbDataAdapter adapter) { }
+        public override string UnquoteIdentifier(string quotedIdentifier) { throw null; }
+        public string UnquoteIdentifier(string quotedIdentifier, System.Data.OleDb.OleDbConnection connection) { throw null; }
+    }
+    public sealed partial class OleDbConnection : System.Data.Common.DbConnection, System.Data.IDbConnection, System.ICloneable, System.IDisposable
+    {
+        public OleDbConnection() { }
+        public OleDbConnection(string connectionString) { }
+        public override string ConnectionString { get { throw null; } set { } }
+        public override int ConnectionTimeout { get { throw null; } }
+        public override string Database { get { throw null; } }
+        public override string DataSource { get { throw null; } }
+        public string Provider { get { throw null; } }
+        public override string ServerVersion { get { throw null; } }
+        public override System.Data.ConnectionState State { get { throw null; } }
+        public event System.Data.OleDb.OleDbInfoMessageEventHandler InfoMessage { add { } remove { } }
+        protected override System.Data.Common.DbTransaction BeginDbTransaction(System.Data.IsolationLevel isolationLevel) { throw null; }
+        public new System.Data.OleDb.OleDbTransaction BeginTransaction() { throw null; }
+        public new System.Data.OleDb.OleDbTransaction BeginTransaction(System.Data.IsolationLevel isolationLevel) { throw null; }
+        public override void ChangeDatabase(string value) { }
+        public override void Close() { }
+        public new System.Data.OleDb.OleDbCommand CreateCommand() { throw null; }
+        protected override System.Data.Common.DbCommand CreateDbCommand() { throw null; }
+        protected override void Dispose(bool disposing) { }
+//TODO  public void EnlistDistributedTransaction(System.EnterpriseServices.ITransaction transaction) { }
```

Comment:
- [ ] public void EnlistDistributedTransaction(System.EnterpriseServices.ITransaction transaction) { }

Reply:
Not sure if this can be made to work. ITransaction is not available in .Net core.

Reply:
Thanks for confirming.